css/about.css

```diff
@@ -0,0 +1,62 @@
+html {
+    max-width: 960px;
+    /* justify-content; */
+    margin: auto;
+    /* margin-left: 30px; */
+}
+
```

Comment:
remember to bring in the font-family to the css for the page. you have them linked in the html but dont specify in the about css that you want to use them. reference your index.css file for how to bring in the font-family to use it on new pages